### PR TITLE
fix(worker): fixed naming convention for az resources. Fixed timeouts…

### LINKED ISSUE #290: Azure resource naming convention improvements
Visit Azure portal and the relevant subscription (DEV)
Notice that it is hard to locate azure resources for given studies

Todo:
Add sanbox name to name of Resource Group
Add Study name to name of Azure Resource

### PR DESCRIPTION
**Pull request by krsteen**
…. Fixed queue handling

Resources now include both study and sandbox names. Timeouts where to short, so queue item got
re-queued and picked up again while it was being processed by another thread.
Closes #290

### CODE REVIEW

File: src/Sepes.Infrastructure/Service/SandboxResourceProvisioningService.cs [hunk 1]
```diff
@@ -74,24 +82,32 @@ public async Task HandleQueueItem(ProvisioningQueueParentDto queueParentItem)
...
-                if (currentResourceOperation != null)
-                {
-                    await _sandboxResourceOperationService.UpdateStatusAndIncreaseTryCount(currentResourceOperation.Id.Value, CloudResourceOperationState.FAILED);
+                    if (currentResourceOperation != null)
+                    {
+                        await _sandboxResourceOperationService.UpdateStatusAndIncreaseTryCount(currentResourceOperation.Id.Value, CloudResourceOperationState.FAILED);
+                    }
+                    else
+                    {
+                        var weird = 1;
```
Comment (HansKristianVikOlsen) [inline src/Sepes.Infrastructure/Service/SandboxResourceProvisioningService.cs]: hæ?

File: src/Sepes.Infrastructure/Util/AzureResourceProivisoningTimeoutResolver.cs [hunk 2]
```diff
@@ -8,23 +8,23 @@ public static int GetTimeoutForOperationInSeconds(string resourceType, string op
...
+                return 180;
             }
             else if (resourceType == AzureResourceType.ResourceGroup)
             {
                 return 30;
             }
             else if (resourceType == AzureResourceType.Bastion)
             {
-                return 300;
+                return 600;
```
Comment (HansKristianVikOlsen) [inline src/Sepes.Infrastructure/Util/AzureResourceProivisoningTimeoutResolver.cs:27]: Does these values just come from testing?

**Review by HansKristianVikOlsen — APPROVED** [review]
No secrets, 10/10. See comments

File: src/Sepes.Infrastructure/Service/SandboxResourceProvisioningService.cs (hunk 1 above)
Comment (krsteen) [reply src/Sepes.Infrastructure/Service/SandboxResourceProvisioningService.cs]: Crap..

File: src/Sepes.Infrastructure/Util/AzureResourceProivisoningTimeoutResolver.cs (hunk 2 above)
Comment (krsteen) [reply src/Sepes.Infrastructure/Util/AzureResourceProivisoningTimeoutResolver.cs:27]: Yes, just observations on how long these operations tend to take. I saw that for Bastion, 300(5 mins) was sometimes to short, so de queue item became visible again. This caused it to be processed by another thread while the old thread was still on it

**Pull request merged**